contribs/gmf/src/controllers/AbstractDesktopController.js

```diff
@@ -5,11 +5,13 @@ goog.require('gmf.controllers.AbstractAppController');
 goog.require('gmf.contextualdata.module');
 goog.require('gmf.drawing.module');
 goog.require('gmf.editing.module');
+goog.require('gmf.import.module'); // only for desktop_alt
```

Comment:
Than is should be in desktop alt, no?

Reply:
Yes but desktop_alt is not a module for now.
But you're right, I will remove it from here

Reply:
... :-/ effectively finally it should be, as the desktop one :-)